app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -93,7 +93,7 @@ const StatusContent = React.createClass({
     const { status } = this.props;
     const { hidden } = this.state;
 
-    const content = { __html: emojify(status.get('content')) };
+    const content = { __html: emojify(status.get('content').replace(/\s*(<p>|<\/p>)\s*/g, '$1').replace(/\s{2}/g, ' &nbsp;')) };
```

Comment:
turning every space into `&nbsp;` will prevent lines wrapping, but I do see what you mean with public timelines. I can add space formatting this to the PR in formatter.rb if that's okay with you, it's less brittle than using `pre-wrap` which will convert a string like `html <br /> string` to

```
html 
…string
```

instead of what's expected.